NucleoTools/modules/dopskript.py

```diff
@@ -0,0 +1,40 @@
+def run_dna_rna_tools(*args):
+    *seqs, action = args
+    if action == "transcribe":
+        result = transcribe(seqs)
+        return "".join(result) if len(seqs) == 1 else result
+    elif action == "reverse":
+        result = reverse(seqs)
+        return "".join(result) if len(seqs) == 1 else result
+    elif action == "complement":
+        result = complement(seqs)
+        return "".join(result) if len(seqs) == 1 else result
+    elif action == "reverse_complement":
+        result = reverse_complement(seqs)
+        return "".join(result) if len(seqs) == 1 else result
+
+def transcribe(seqs):
+    result = []
+    for i in seqs:
+        if "U" in i or "u" in i:
+            result.append("Введите ДНК")
+        else:
+            result.append(i.replace("T", "U").replace("t", "u"))
+    return result

```

Comment:
```suggestion
def transcribe(seq):
    if "U" in seq or "u" in seq:
        return seq.replace('T', 'U').replace('t', 'u'))
    else:
        return ("Введите ДНК")  # Здесь как будто надо написать (incorrect input или что-то такое)
```

С одной стороны прикольно, как у тебя получилось структурировать  это все в run_dna_rna_tools.
С другой стороны, мне кажется немного расточительным вызывать цикл внутри дополнительных функций. Как будто лучше сначала сделать проверку  на неподался ли мне один  сиквенс(в основной функции)?  Чем все время вызывать цикл, вне зависимости от len(seqs). 
Но если оставлять так, то i лушче заменить на seq.